README.md

```diff
@@ -94,27 +94,16 @@ cd snowflake-connector-nodejs
 npm install .
 ```
 
-3. Install the nyc module which displays the code coverage
+3. Run tests
 ```
-npm install nyc
-```
-
-4. Edit the package.json file and replace the specified test with the one below:
-```
-"scripts": {
-  "test": "nyc mocha test/**/*.js"
-},
+npm test
```

Comment:
this script should add coverage part somehow included - we don't want to calculate coverage on each test run

Reply:
Ok I've created separated task.

Reply:
ok

Reply:
and here we don't need to run `test` since below is `test:coverage`

Reply:
removed